content-full-no.php

```diff
@@ -11,16 +11,16 @@
 
 		<div class="content-page">
 		
-			<?php if (has_post_thumbnail()): ?>
+			<?php if ( has_post_thumbnail() ) : ?>
 			<div class="featuredImage">
-				<?php echo the_post_thumbnail(700, 300); ?>
+				<?php echo the_post_thumbnail( 700, 300 ); ?>
 			
 			</div>	
 			<?php endif; ?>
 			
 			
 			<div class="entry-content">
-				<?php if (!$isRoot): ?>
+				<?php if ( ! $isRoot ) : ?>
```

Comment:
Let me look up the standard here - I want to say the space between the operator and the term is, in fact, required.